components/widgets/admin_console/admin_panel.tsx

```diff
@@ -7,7 +7,20 @@ import {FormattedMessage} from 'react-intl';
 
 import FormattedMarkdownMessage from 'components/formatted_markdown_message';
 
-const AdminPanel = (props) => (
+type Props = {
+    id?: string,
+    className: string,
+    onHeaderClick?: React.EventHandler<React.MouseEvent>,
+    titleId: string,
+    titleDefault: string,
+    subtitleId: string,
+    subtitleDefault: string,
+    subtitleValues?: any,
+    button?: React.ReactNode,
+    children?: React.ReactNode,
+};
+
+const AdminPanel = (props: Props) => (
```

Comment:
I think this should be: `const AdminPanel: React.FC<Props> = (props: Props) => (` to get proper code completion